src/main/java/com/stuypulse/robot/util/ReflectiveTapeData.java

```diff
@@ -0,0 +1,46 @@
+/************************ PROJECT JIM *************************/
+/* Copyright (c) 2023 StuyPulse Robotics. All rights reserved.*/
+/* This work is licensed under the terms of the MIT license.  */
+/**************************************************************/
+
+package com.stuypulse.robot.util;
+
+import com.stuypulse.stuylib.math.Angle;
+import com.stuypulse.stuylib.network.limelight.Limelight;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Pose3d;
+
+public class ReflectiveTapeData extends VisionData {
+
+    public ReflectiveTapeData(Pose2d robotPose, double latency, int id, Pose3d cameraPose) {
+        super(robotPose, latency, cameraPose);
+    }
+
+    public boolean hasAnyTarget() {
+        return Limelight.getInstance().getValidTarget();
+    }
+    
```

Comment:
Maybe override the getPose function and just return a pose2d from the distance and angle you get from the reflective tape